src/parcsr_ls/HYPRE_parcsr_int.c

```diff
@@ -34,6 +34,8 @@ hypre_ParReadVector( MPI_Comm comm, const char* file )
 
 HYPRE_Int hypre_ParVectorSize(void * x)
 {
+   HYPRE_UNUSED_VAR(x);
+
```

Comment:
This is another strange routine. Why does this even exist?

Reply:
True, this file seems to contain several incomplete functions dated back to the LOBPCG work.